examples/sqoop-parquet-full-load/tables.yml

```diff
@@ -26,8 +26,10 @@ tables:
     comment: "Table comment" # Table comment
     metadata:
        LOAD_FREQUENCY: Daily
-       CONTACT: developer@company.io
+       CONTACT_INFO: developer@company.io
        SOURCE: employee database
+       SECURITY_CLASSIFICATION: Public
+       CDP_MANAGED: "TRUE"
```

Comment:
`SECURITY_CLASSIFICATION` and `CDP_MANGED` are likely not great candidates for public Pipewrench given Cargill context.  They should be added to internal fork.